docs/organization.rst

```diff
@@ -0,0 +1,54 @@
+==================================
+Project Governance
+==================================
+
+This document informally outlines the organizational structure governing the
+Planemo code base hosted at https://github.com/galaxyproject/planemo. This
+governance extends to code-related activities of this repository such as
+releases and packaging and related Planemo projects such planemo-machine. This
+governance does not include any other Galaxy-related projects belonging to the
+``galaxyproject`` organization on GitHub.
+
+Benevolent Dictator for Now (BDFN)
+===================================
+
+John Chilton (@jmchilton) is the benevolent dictator for now (BDFN) and is solely
+responsible for setting project policy. The BDFN is responsible for maintaining
+the trust of the developer community and so should be consistent and
+transparent in decision making processes and request comment and build
+consensus whenever possible.
+
+The BDFN position does not exist because there is not a desire to have open
```

Comment:
How about:
`
The BDFN position only exists because the developers of the
project believe it is currently too small to support full and open governance at this
time. In order to ...
`
Less double negatives, shorter, easier to understand?